app/main.py

```diff
@@ -1 +1,38 @@
-#  write your code here
+class Animal:
+    def __init__(self, name, appetite, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+    def __init__(self, name, is_hungry=True):
+        super(Cat, self).__init__(name, 3, is_hungry)
```

Comment:
It is not clear what "3" is, it is better to indicate the name of the variable to which we assign this value.